src/algebra/ring.lean

```diff
@@ -220,6 +220,8 @@ def nonzero_comm_semiring.of_ne [comm_semiring α] {x y : α} (h : x ≠ y) : no
   zero_ne_one := λ h01, h $ by rw [← one_mul x, ← one_mul y, ← h01, zero_mul, zero_mul],
   ..show comm_semiring α, by apply_instance }
 
+def has_div_of_filter_product [division_ring α] : has_div α := division_ring_has_div
```

Comment:
I don't understand this change. At a minimum the name is not right (there is nothing about filters here), but why is the change necessary at all?

Reply:
Lean 3.5.0c changed the instance `division_ring_has_div` because it assumed the same instance twice. I can rename this, I didn't notice the problem before.